server/src/main/java/org/elasticsearch/reservedstate/service/ReservedClusterStateService.java

```diff
@@ -423,7 +423,7 @@ public void process(
             ErrorState errorState = new ErrorState(
                 projectId,
                 namespace,
-                reservedStateChunks.getFirst().metadata().version(),
+                reservedStateChunks.isEmpty() ? null : reservedStateChunks.getFirst().metadata().version(),
```

Comment:
I think we should use EMPTY_VERSION?

https://github.com/elastic/elasticsearch/blob/eceb178cd73dce7f78021d6859b7ac74ea2d7431/server/src/main/java/org/elasticsearch/cluster/metadata/ReservedStateMetadata.java#L51